ext/afform/admin/ang/afGuiEditor/config-form.html

```diff
@@ -84,5 +84,16 @@
       <p class="help-block">{{:: ts('Choose which contact from the search should match the contact being viewed.') }}</p>
     </div>
   </fieldset>
+  <fieldset>
+
+    <legend>{{:: ts('Form Actions') }}</legend>
 
+    <div class="form-group" ng-if="afform.submit_action && afform.submit_action == 'redirect'" ng-class="{'has-error': !!config_form.redirect.$error.pattern}">
```

Comment:
@seamuslee001 I tried running this but couldn't view the field. I think due to this:

```suggestion
    <div class="form-group" ng-class="{'has-error': !!config_form.redirect.$error.pattern}">
```

Reply:
Agreed fixed now